docs/source/pipelines.mdx

```diff
@@ -12,12 +12,12 @@ specific language governing permissions and limitations under the License.
 
 # Optimum pipelines for inference
 
-The [`pipeline`] makes it simple to use models from the [Model Hub](https://huggingface.co/models) for accelerated inference on a variety of tasks such as text classification.
-Even if you don't have experience with a specific modality or understand the code powering the models, you can still use them with the [`pipeline`]! This tutorial will teach you to:
+The [`~pipelines.pipeline`] makes it simple to use models from the [Model Hub](https://huggingface.co/models) for accelerated inference on a variety of tasks such as text classification.
+Even if you don't have experience with a specific modality or understand the code powering the models, you can still use them with the [`~pipelines.pipeline`]! This tutorial will teach you to:
 
 <Tip>
 
-You can also use the `pipeline()` function from Transformers and provide your `OptimumModel`.
+You can also use the [pipeline](https://huggingface.co/docs/transformers/main/en/main_classes/pipelines#pipelines) function from Transformers and provide your `OptimumModel`.
```

Comment:
turbo nit for consistency and wasn't sure if `OptimumModel` is a well-defined object:
```suggestion
You can also use the [pipeline()](https://huggingface.co/docs/transformers/main/en/main_classes/pipelines#pipelines) function from Transformers and provide your Optimum model class.
```